### PR TITLE
fix: prefer apt over brew

### PR DESCRIPTION
**Pull request by mathnogueira**
This PR fixes docker-compose installation on Linux when brew is available.

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: cli/installer/util.go [hunk 1]
```diff
@@ -63,10 +63,10 @@ func detectPkgManager() pkgManager {
 	}
 
 	switch true {
+	case commandExists("apt"):
```
Comment (mathnogueira) [inline cli/installer/util.go:66]: apt-get is deprecated

Comment (schoren) [reply cli/installer/util.go:66]: it's not deprecated. it's intended for scripting, while `apt` is intended for manual user interaction. https://askubuntu.com/a/829866

https://manpages.ubuntu.com/manpages/xenial/man8/apt.8.html
https://linux.die.net/man/8/apt-get

**Review by schoren — CHANGES_REQUESTED** [review]
please revert back to using apt-get. the order looks fine

File: cli/installer/util.go [hunk 2]
```diff
@@ -63,10 +63,10 @@ func detectPkgManager() pkgManager {
 	}
 
 	switch true {
-	case commandExists("brew"):
-		lastDetectedPkgManager = homebrew
 	case commandExists("apt-get"):
 		lastDetectedPkgManager = apt
+	case commandExists("brew"):
```
Comment (schoren) [inline cli/installer/util.go:68]: I assume the same problems wuold happen on any linux with brew, so maybe it's better to move it below all linux package managers?

**Pull request merged**